### PR TITLE
doc: update connector tutorials to reference airbyte-ci for connector build

### LINKED ISSUE #31648: update connector tutorial with airbyte-ci reference

### PR DESCRIPTION
**Pull request by alafanechere**
## What
Closes #31648
Update our tutorial to reference `airbyte-ci` for connector build.

### CODE REVIEW

**Comment by alafanechere** [comment]
Current dependencies on/for this PR:
* master
  * **PR #31650** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/31650" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/31650?utm_source=stack-comment).

File: docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md [hunk 1]
```diff
@@ -32,8 +32,38 @@ _Note: Not all types of tests work for all connectors, only configure the ones t
...
+It will not use a Dockerfile but will build the connector image from our [base image](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/base_images/README.md) and use our internal build logic to build an image from your Python connector code.
+
+Running `airbyte-ci connectors --name source-<source-name> build` will build your connector image.
+Once the command is done, you will find your connector image in your local docker host: `airbyte/source-<source-name>:dev`.
+
+
+
+**Option B: Building the docker image with a Dockerfile**
+
+If you don't want to rely on `airbyte-ci` to build your connector, you can build the docker image using your own Dockerfile.
```
Comment (evantahler) [inline docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md:47]:
```suggestion
If you don't want to rely on `airbyte-ci` to build your connector, you can build the docker image using your own Dockerfile.  This method is not preferred, and is not supported for certified connectors
```

File: docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md [hunk 2]
```diff
@@ -32,8 +32,38 @@ _Note: Not all types of tests work for all connectors, only configure the ones t
 
 Build your connector image if needed.
 
-```text
-docker build .
+**Option A: Building the docker image with `airbyte-ci`**
+
```
Comment (evantahler) [inline docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md:36]:
```suggestion

This is the preferred method for building and testing connectors.

```

File: docs/connector-development/tutorials/building-a-python-source.md [hunk 3]
```diff
@@ -128,22 +128,55 @@ python main.py read --config secrets/config.json --catalog sample_files/configur
...
+It will not use a Dockerfile but will build the connector image from our [base image](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/base_images/README.md) and use our internal build logic to build an image from your Python connector code.
+
+Running `airbyte-ci connectors --name source-<source-name> build` will build your connector image.
+Once the command is done, you will find your connector image in your local docker host: `airbyte/source-<source-name>:dev`.
+
+
+
+**Option B: Building the docker image with a Dockerfile**
+
+If you don't want to rely on `airbyte-ci` to build your connector, you can build the docker image using your own Dockerfile.
```
Comment (evantahler) [inline docs/connector-development/tutorials/building-a-python-source.md:148]:
```suggestion
If you don't want to rely on `airbyte-ci` to build your connector, you can build the docker image using your own Dockerfile.  This method is not preferred, and is not supported for certified connectors
```

File: docs/connector-development/tutorials/cdk-speedrun.md [hunk 4]
```diff
@@ -231,10 +231,42 @@ python main.py read --config sample_files/config.json --catalog sample_files/con
...
+It will not use a Dockerfile but will build the connector image from our [base image](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/base_images/README.md) and use our internal build logic to build an image from your Python connector code.
+
+Running `airbyte-ci connectors --name source-<source-name> build` will build your connector image.
+Once the command is done, you will find your connector image in your local docker host: `airbyte/source-<source-name>:dev`.
+
+
+
+**Option B: Building the docker image with a Dockerfile**
+
+If you don't want to rely on `airbyte-ci` to build your connector, you can build the docker image using your own Dockerfile.
```
Comment (evantahler) [inline docs/connector-development/tutorials/cdk-speedrun.md:247]:
```suggestion
If you don't want to rely on `airbyte-ci` to build your connector, you can build the docker image using your own Dockerfile.  This method is not preferred, and is not supported for certified connectors
```

**Review by evantahler — APPROVED** [review]

**Review by marcosmarxm — APPROVED** [review]
Thanks @alafanechere

**Pull request merged**